libass/ass_blur.c

```diff
@@ -162,7 +162,7 @@ static void find_best_method(BlurMethod *blur, double r2)
         calc_coeff(mu, blur->radius, r2, mul);
     }
     for (int i = 0; i < blur->radius; i++)
-        blur->coeff[i] = (int) (0x10000 * mu[i] + 0.5);
+        blur->coeff[i] = ass_lrint(0x10000 * mu[i]);
```

Comment:
cc @MrSmile

(btw would I be right to assume `mu[i]` are always non-negative?)

Reply:
Yes, it's explicitly enforced [here](https://github.com/libass/libass/blob/master/libass/ass_blur.c#L139). I've used `(int)` as it generates optimal code more consistently than `lrint`, but it probably doesn't matter much here.